docs/rules.rst

```diff
@@ -26,8 +26,9 @@ a winner announced in each category.
 All members of a team get to vote (though not for their own team), enter diary
 entries and upload files. People may join more than one team.
 
-During the challenge, competitors are encouraged to hang out on IRC and
-post diary entries (diaries are supplied as part of the challenge).
+During the challenge, competitors are encouraged to post diary entries (diaries
+are supplied as part of the challenge), and hang out in
```

Comment:
```suggestion
are supplied as part of the challenge), and hang out on
```

I think "in the Discord" or "on Discord" are acceptable, but "in Discord" looks very weird to me.